src/services/ApiClient.js

```diff
@@ -0,0 +1,29 @@
+import axios from "axios";
+
+const URL = "http://testscrud-env.eba-fpb5kcf8.us-east-1.elasticbeanstalk.com";
+
+function logError(errorResponse) {
+  const response = errorResponse.response;
+
+  if (response && response.data && response.data.error) {
```

Comment:
I wonder if this is where the 'optional' chaining operator could be used? I have always wanted to use it... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining idk tho?

Reply:
never seen this one before! I can play around and see if it works